src/concourse/pipelines/container_images/openedx_tubular.py

```diff
@@ -14,8 +14,8 @@
 
 tubular_repository = git_repo(
     name=Identifier("openedx-tubular"),
-    uri="https://github.com/openedx/tubular",
-    branch="master",
+    uri="https://github.com/mitodl/tubular",
```

Comment:
Maybe just add a TODO comment here so we can grep for it.

Reply:
I pulled this change out of this PR and made it under its own. Good idea though.